lib/remote/modifyobjecthandler.cpp

```diff
@@ -64,7 +64,7 @@ bool ModifyObjectHandler::HandleRequest(
 
 	Value attrsVal = params->Get("attrs");
 
-	if (attrsVal.GetReflectionType() != Dictionary::TypeInstance) {
+	if (attrsVal.GetReflectionType() != Dictionary::TypeInstance && attrsVal.GetType() != ValueEmpty) {
 		HttpUtility::SendJsonError(response, params, 400,
 			"Invalid type for 'attrs' attribute specified. Dictionary type is required."
 			"Or is this a POST query and you missed adding a 'X-HTTP-Method-Override: GET' header?");
```

Comment:
AFAIK you already succeed despite an empty dict of attrs. I don’t see much difference to this case.

Reply:
It's basically the question in which situations would "Or is this a POST query and you missed adding a 'X-HTTP-Method-Override: GET' header?" be a useful hint.

When you sent something like `{"attrs": ["address"]}` it definitely is because that would be valid for a GET request. In this case, you would still get that hint.

But if you include neither `attrs` nor `restore_attrs` in your request, I can imagine many situations where that's certainly not what you want to do and some hint might be helpful, the best example I could come up with would be if your request body is `{"vars.os": "Linux"}` (i.e. you forgot to wrap it in `{"attrs": ...}`), you will have great fun debugging that if the API responds with "Attributes updated." to that request. At the moment, this returns an error.